src/App.tsx

```diff
@@ -50,15 +36,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector value={author} />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && !loading && <Loader />}
```

Comment:
The condition for displaying the `Loader` seems inverted. It should be shown when `loading` is true, not false. Consider changing the condition to `{author && loading && <Loader />}`.